app/models/grda_warehouse/tasks/chronically_homeless.rb

```diff
@@ -356,7 +356,7 @@ def homeless_in_all_last_12_months? months:
     end
 
     def disabled?(client_id)
-      @disabled_clients ||= GrdaWarehouse::Hud::Client.where(id: @clients).disabled_client_ids
+      @disabled_clients ||= GrdaWarehouse::Hud::Client.where(id: @clients).chronically_disabled
```

Comment:
There is a comment at the top about the definition of chronic that probably needs to be changed.

Reply:
I *believe* that was HUD's language.  It's what lead to the confusion all along.  I'm going to leave it, as it is technically true, and the code now documents how it does the calculation.